WordPress/Resources/AppStoreStrings.po

```diff
@@ -15,8 +15,7 @@ msgstr ""
 
 #. translators: Subtitle to be displayed below the application name in the Apple App Store. Limit to 30 characters including spaces and commas!
 msgctxt "app_store_subtitle"
-msgid "Build a website or blog
-"
+msgid "Build a website or blog"
```

Comment:
I'm not sure why GitHub is showing this change. It comes from c72d2078b31acce0a0c1c91d8f6d10dfa888d7d6, which has already been merged into `develop` in #16696